lib/errors.ts

```diff
@@ -120,6 +120,14 @@ export enum ConditionErrorNameMapper {
    * Error is thrown when the service is unavailable. The operation should be retried.
    */
   "com.microsoft:timeout" = "ServiceUnavailableError", // Retryable
+  /**
+   * Error is thrown when no new meesages are received for the specified time.
+   */
+  "com.microsoft:message-wait-timeout" = "MessageWaitTimeout",
+  /**
+   * Error is thrown when timeout happens for the said operation.
+   */
+  "amqp:operation-timeout" = "OperationTimeoutError",
```

Comment:
These are existing errors that we are now exposing, correct?

Reply:
Nope. I have made them up by giving them logical and meaningful names. This makes it easier to translate them and whether they need to be retried or not.

Reply:
I know `OperationTimeoutError` is the made up name :) What I meant to ask was if `amqp:operation-timeout` is an existing error.

Reply:
I see. Nope, the condition is made up as well. Gave some logical name to the condition.

Reply:
Started the condition with amqp: v/s com.microsoft: because this error is thrown from rhea-promise when the promise timeout happens. This will be useful to other folks consuming rhea-promise.